source/engineClass.py

```diff
@@ -1051,6 +1057,15 @@ def getHFInputs(self,hfMode,
         self.HF.qFilePath = qFilePath
         self.HF.qFileTag = qFileTag
 
+        if rzqFiledata != None:
+            print("saving rzq file to tmpDir.")
+            self.HF.rzqFile = self.HF.writerzqFileData(rzqFile, rzqFiledata, self.tmpDir)
+        else:
+            print(rzqFile)
```

Comment:
What is this print statement doing?  If it is necessary, please add string so users know what you are printing.